packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -262,6 +264,35 @@ function TokenRow({ style, onClick, token }: TokenRowProps): JSX.Element {
     )
   }, [isDepositMode, l1Network, l2Network])
 
+  const tokenBalanceContent = useMemo(() => {
+    if (!tokenIsAddedToTheBridge) {
+      return <span className="text-sm font-medium text-blue-link">Import</span>
+    }
+
+    function wrapContent(content: React.ReactNode) {
+      return (
+        <span className="flex items-center whitespace-nowrap text-sm text-gray-500">
+          {content}
+        </span>
+      )
+    }
```

Comment:
I'm not sure we need a function for that, but if we want go that way, we should create a component instead
```suggestion
    function wrapContent({ children }: React.PropsWithChildren) {
      return (
        <span className="flex items-center whitespace-nowrap text-sm text-gray-500">
          {children}
        </span>
      )
    }
```

Reply:
Used ternary instead